src/constants/user.js

```diff
@@ -0,0 +1,104 @@
+import axios from "axios"
+import {baseUrl} from "./constants"
+import { goToFeedPage, goToAdressPage, goToSignUpPage, goToProfilePage } from "../router/coordinator"
+
+
+const axiosConfig = {
+  headers: {
+    auth: localStorage.getItem('token')
+  }
+}
```

Comment:
Isolar a axiosConfig dessa forma, nesse caso, não é uma boa. Nos casos em que a configuração do headers necessita de um valor do localStorage, que pode mudar com o tempo, é melhor isolar tudo em uma função e chamar a função sempre que for usar a axiosConfig, dessa forma:


const generateAxiosConfig = () =>{
     const axiosConfig = {
         headers: { auth: window.localStorage.getItem("token") }
     }

    return axiosConfig
}